data/mods/Magiclysm/Spells/attunements/Alchemist.json

```diff
@@ -0,0 +1,82 @@
+[
+  {
+    "id": "alchemist_make_potion_bomb",
+    "type": "SPELL",
+    "name": "Manafusion bomb",
+    "description": "Oversaturates potion starter with mana to create a deadly mana explosion.",
+    "valid_targets": [ "none" ],
+    "flags": [ "SOMATIC", "WONDER" ],
```

Comment:
wonder? i don't think so

Reply:
Wonder is for spawning more than 1 bomb per cast